src/types/settings.ts

```diff
@@ -12,6 +12,7 @@ export type GeneralSettings = {
   badges: boolean;
   autoApplyQuickCss: boolean;
   showWelcomeNoticeOnOpen: boolean;
+  reactDevTools: boolean;
 };
 
 export const defaultSettings: Partial<GeneralSettings> = {
```

Comment:
Unrelated to this PR but the type here should be using `satisfies`:

```ts
export const defaultSettings = {
  badges: true,
  autoApplyQuickCss: false,
  showWelcomeNoticeOnOpen: true,
  reactDevTools: false,
} satisfies Partial<GeneralSettings>;
```
GH won't let me suggest here but can you fix it in this branch?